src/main/java/billgates/view/gui/SignInDialog.java

```diff
@@ -0,0 +1,110 @@
+package billgates.view.gui;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.Arrays;
+
+public class SignInDialog extends JDialog {
+        JTextField nameInput = new JTextField(17);
+        JPasswordField passwordInput = new JPasswordField(17);
```

Comment:
According to [Improvement 1] [here](https://github.com/CSC207-2022F-UofT/course-project-bill-gates/issues/22)
We should have a maximum username length of 10 characters and a maximum password length of 16 characters. 

Maybe we can implement it here as well?